crafter-search-server/src/main/resources/crafter/search/services-context.xml

```diff
@@ -151,6 +151,15 @@
                value="${crafter.search.solr.field.type.datetime.multiValued.suffix}"/>
     </util:map>
 
+    <bean id="corsFilter" class="org.craftercms.commons.web.CORSFilter">
```

Comment:
Please name the bean `crafter.corsFilter` to be consistent with the other bean names.